docs/csharp/whats-new/tutorials/snippets/staticinterfaces/Utilities.cs

```diff
@@ -0,0 +1,7 @@
+﻿public static class Utilities
+{
+    // <MidPoint>
+    public static T MidPoint<T>(T left, T right)
+        where T : INumber<T> => (left + right) / (T.One + T.One);
```

Comment:
I know this is just an example, but it may suffer from overflow.
Should this be written to avoid overflow*? But it complicates the example, so maybe just
```suggestion
        where T : INumber<T> => (left + right) / (T.One + T.One);  // note: the addition of left, and right may overflow here, it's just for demonstration purposes
```
Hm, I don't like...but the docs should guide people to good and robust code, so somehow this should be mentionend.

Or use a different example than _Midpoint_.

\* for instance: right + (left - right) / 2